README.md

```diff
@@ -21,12 +21,12 @@ There are six functions in this package, and their descriptions are provided in
 
 ```S
 ans = mrs(X, G)
-ans = mrs_nested(X, G, H)
+ans = andova(X, G, H)
 summary(ans)
 plot1D(ans)
 plot2D(ans)
 plotTree(ans)
 ```
 
 ### Reference
-Soriano J. and Ma L. (2014). Multi-resolution two-sample comparison through the divide-merge Markov tree. (http://arxiv.org/abs/1404.3753)
\ No newline at end of file
+Soriano J. and Ma L. (2015). Probabilistic multi-resolution scanning for two-sample differences. JRSS-B (To appear).
```

Comment:
Add also Analysis of distributional variation through multi-scale Beta-Binomial modeling reference?